tests/examples/CMakeLists.txt

```diff
@@ -188,6 +188,14 @@ function(run_test test_path namespace escaped_content)
         set(DOC_BLOCK_PREFIX "@usage")
     endif()
 
+    # Does the text generate text output?
+    if(tmp_content MATCHES "--DOC_GEN_OUTPUT")
+        # The expected output is next to the .lua file in a .txt file
+        string(REPLACE ".lua" ".txt" expected_output_path ${test_path})
```

Comment:
Should this maybe use a more distinct extension, e.g. appending `.output.txt` (or just `.output`) to the file name?

Reply:
I don't know. I used a `.txt` extension to make it clearer that this is no fancy binary format. I do not really care about the color of the bikeshed here.

What do others say?

Reply:
My point is that it easier / more specific to find `*.output.txt` instead of `*.txt` when looking for generated output.

Reply:
I don't think it's really worth arguing about and can be changed later. `.txt` is fine, `.output.txt` is better. I don't think it should block this PR.

Reply:
Sure, but once it is in there is at least overhead of doing a 2nd PR, and nobody might care then anyway anymore - that's what reviews are for.. ;)

Reply:
Done now, thanks @psychon!